pkg/handlers/logout.go

```diff
@@ -9,12 +9,20 @@ import (
 )
 
 func (h *Handler) Logout(w http.ResponseWriter, r *http.Request) {
-	if r.Header.Get("Authorization") == "" {
-		w.WriteHeader(http.StatusNoContent)
-		return
+
+	auth := r.Header.Get("authorization")
```

Comment:
do we need to check the `Authorization` header if we're sending the token in a cookie?

Reply:
Since this is only used by the UI it is safe to remove, I went ahead and removed it now.